components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit-finish-local-ajaxprocessor.jsp

```diff
@@ -40,6 +40,12 @@
         return;
     }
 
+    if(session.isNew()){
+        response.sendRedirect("../admin/login_action.jsp");
```

Comment:
Should not this be handled in generic way in a valve or something. As I see all AJAX requests have this problem.